pkgs/top-level/all-packages.nix

```diff
@@ -16739,6 +16739,8 @@ in
 
   rspamd = callPackage ../servers/mail/rspamd { };
 
+  smtp-to-telegram = callPackage ../servers/mail/smtp-to-telegram { };
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
I'm not sure what you mean - care to elaborate?

Reply:
If all packages would be added to the end of the file we would have a lot of merge conflicts. The file has categories separated by comments. You find a fitting one and sort the package their alphabetically.

Reply:
Hmm, then as far as I understand it, it's alright, isn't it? It's inside the `### SERVERS` category ( ✅ ), right after `rspamd` (✅ ).